backend/src/test/java/com/htecgroup/skynest/service/impl/FileServiceImplTest.java

```diff
@@ -112,6 +116,37 @@ void getAllFilesWithParent() {
         .findAllByParentFolderId(fileMetadataEntity.getParentFolder().getId());
   }
 
+  @Test
+  void moveFileToFolderTest() {
```

Comment:
Can you pls take a look at the test naming that @MetoMladenovski uses and try to follow it?